src/Xamarin.Android.Build.Tasks/Tasks/GetAdditionalResourcesFromAssemblies.cs

```
@@ -321,7 +322,15 @@ string MakeSureLibraryIsInPlace (string destinationBase, string url, string vers
 					LogDebugMessage ("    found `{0}` in `{1}`", embeddedArchive, Path.Combine (AndroidSdkDirectory, extraPath));
 			}
 
-			string contentDir = string.IsNullOrEmpty (extraPath) ? Path.Combine (destinationDir, "content") : Path.Combine (AndroidSdkDirectory, extraPath);
+			string oldcontentDir = Path.Combine (destinationDir, "content");
+			if (Directory.Exists (oldcontentDir)) {
+				try {
+					Directory.Delete (oldcontentDir, recursive: true);
+				}
+				catch {
```

Please don't just swallow exceptions. *Log* the errors.